composer.json

```diff
@@ -11,10 +11,12 @@
             "email": "dcramer@gmail.com"
         }
     ],
+    "minimum-stability": "dev",
     "require-dev": {
         "friendsofphp/php-cs-fixer": "~2.1",
         "phpunit/phpunit": "^4.8 || ^5.0",
-        "monolog/monolog": "*"
+        "nokitakaze/testhttpserver": "^0.1.0",
```

Comment:
I know Python contains some serious thing like Tornado. But **I don\'t know Python** (otherwise I used it for testing purpose) and async/outer curl **should be coveraged** so I decide to write my own testing server. If anybody could do codecoverage this Python\'s HTTP server please do it that way and I will close this PR.